src/Container.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Container = ({ children, isFinished, animationDuration }) => (
+  <div
+    style={{
+      opacity: isFinished ? 0 : 1,
+      pointerEvents: 'none',
+      transition: `opacity ${animationDuration}ms linear`,
+    }}
+  >
+    {children}
+  </div>
+);
+
+export default Container;
```

Comment:
Can we move this file out of root and into the components folder? A more descriptive name may also be helpful.